Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 81 fix the content type according to requirement

## Related ticket:
[SCRUM-81](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-81?atlOrigin=eyJpIjoiMzdlNTA5ZTE2ZjJhNDEzZWE0MTkyMmZjNTFiMjFiZDIiLCJwIjoiaiJ9)

## In this PR:
- Fix all content types according to client requirements
- Add dummy content for each type

## Testing instruction
- Checkout branch
- Cd to Backend/
- Import db ```lando db-import SCRUM-81.gz```
- Import config: ```lando drush cim```
- Check the Drupal site for all the contents and contents type
